docs/recipes/babelrc.md

```diff
@@ -57,16 +84,17 @@ Set `babelrc` to `true`. This will use your `.babelrc` and extend it with any ad
 }
 ```
```

Comment:
Maybe include `babel` config here to show the source of what's merged for clarity?